systemd/ua-update-contract.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Ubuntu Advantage auto attach
+After=cloud-final.target
```

Comment:
At some point you and I might have to think about systems without cloud-init installed (like bare metal or desktop) and define the right After clauses for them